packages/_server/src/config/cspellConfig.ts

```diff
@@ -599,6 +599,25 @@ interface CSpellSettingsPackageProperties extends CSpellSettings {
 
     /**
      * @scope resource
+     * @description
+     * @markdownDescription
+     * List of RegExp patterns or Pattern names to exclude from spell checking.
+     * The default RegExp flag is `//gi`
+     * 
+     * **GUI Examples**
+     * 
+     * Everyline is a RegExp pattern or a Pattern names - comment
```

Comment:
Here is the list of predefined patterns:
```
-  `Base64`, `Base64MultiLine`, `Base64SingleLine`, `CStyleComment`, `CStyleHexValue`, 
   `CSSHexValue`, `CommitHash`, `CommitHashLink`, `Email`, `EscapeCharacters`, 
   `HexValues`, `href`, `PhpHereDoc`, `PublicKey`, `RsaCert`, `SshRsa`, `SHA`, 
   `HashStrings`, `string`, `UnicodeRef`, `Urls`, `UUID`
```

Reply:
I used `venv`